Gems/Camera/Code/Source/EditorCameraComponent.cpp

```diff
@@ -117,8 +118,12 @@ namespace Camera
                     ->UIElement(AZ::Edit::UIHandlers::Button,"", "Sets the view to this camera")
                         ->Attribute(AZ::Edit::Attributes::ChangeNotify, &EditorCameraComponent::OnPossessCameraButtonClicked)
                         ->Attribute(AZ::Edit::Attributes::ButtonText, &EditorCameraComponent::GetCameraViewButtonText)
-                    ->ClassElement(AZ::Edit::ClassElements::Group, "Debug")
+                    ->UIElement(AZ::Edit::UIHandlers::Button,"", "Sets this camera to view")
+                        ->Attribute(AZ::Edit::Attributes::ChangeNotify, &EditorCameraComponent::OnMatchViewportClicked)
+                        ->Attribute(AZ::Edit::Attributes::ButtonText,  "Match Viewport")
```

Comment:
Perhaps this is intentional? This button text does not match the button text in the description or screenshots which says "Align camera to view".    

Also it is capitalizing both words, but the other buttons and fields only capitalize the first word.